executor.go

```diff
@@ -1618,6 +1618,9 @@ func (e *Executor) translateCall(index string, idx *Index, c *pql.Call) error {
 	// Translate row key, if field is specified & key exists.
 	if fieldName != "" {
 		field := idx.Field(fieldName)
+		if field == nil {
```

Comment:
this and the following block fix a bug introduced by the newpql work where the field could be nil.